operators/src/mock/mock_raster_point_join.rs

```diff
@@ -127,18 +119,34 @@ impl VectorOperator for MockRasterPointJoinOperator {
             _ => panic!(),
         }))
     }
+    fn result_descriptor(&self) -> VectorResultDescriptor {
+        if self.result_descriptor.is_none() {
+            let new_result_descriptor = VectorResultDescriptor {
+                projection: self.point_sources.get(0).map_or_else(
+                    || ProjectionOption::None,
+                    |o| o.result_descriptor().projection,
+                ),
+                data_type: VectorDataType::MultiPoint,
+            };
+
+            // self.result_descriptor = Some(new_result_descriptor); //TODO: discuss if this should be interior or if method is mut...
```

Comment:
to enable storage of the computed information, we need to change the result_descriptor method take &mut self. Is there any reason to not do this?